keyboards/dztech/tofu/jr/v1/keymaps/vial/vial.json

```
@@ -0,0 +1,165 @@
+{
+  "name": "TOFU_JR",
+  "vendorId": "0x445A",
+  "productId": "0x1426",
+  "matrix": {
+    "rows": 5,
+    "cols": 15
+  },
+  "lighting": "vialrgb",
+  "keycodes": [
+    "qmk_lighting"
+  ],
+  "menus": [
+    "qmk_rgb_matrix"
+  ],
```

```suggestion
  "keycodes": [
    "qmk_lighting"
  ],
  "menus": [
    "qmk_rgb_matrix"
  ],
```
Vial does not recognize these objects, so they can just be deleted.